Userland/Libraries/LibPDF/ColorSpace.cpp

```diff
@@ -39,8 +39,6 @@ PDFErrorOr<NonnullRefPtr<ColorSpace>> ColorSpace::create(DeprecatedFlyString con
         return DeviceRGBColorSpace::the();
     if (name == CommonNames::DeviceCMYK)
         return DeviceCMYKColorSpace::the();
-    if (name == CommonNames::Pattern)
-        return Error::rendering_unsupported_error("Pattern color spaces not yet implemented");
```

Comment:
This up here was unreachable as far as I can tell: ColorSpace.h has:

```
#define ENUMERATE_COLOR_SPACE_FAMILIES(V) \
...
    V(Pattern, false)                     \
...
```

where the second arg is `never_needs_parameters()`, and Renderer.cpp has:

```
        if (color_space_family.never_needs_parameters()) {
            return ColorSpace::create(color_space_name);
        }
```

So this factory function was never called for Pattern spaces; instead the factory function below taking arguments is called.